src/main/kotlin/com/demonwav/mcdev/util/McPsiClass.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+@file:JvmName("McPsiClass")
+package com.demonwav.mcdev.util
+
+import com.intellij.navigation.AnonymousElementProvider
+import com.intellij.openapi.extensions.Extensions
+import com.intellij.psi.PsiClass
+import com.intellij.psi.PsiClassType
+import com.intellij.psi.PsiElement
+
+// Type
+
+internal val PsiClassType.fullQualifiedName
+    get() = resolve()!!.fullQualifiedName
```

Comment:
I want to add that some of the assert non-null calls here are actually correct. For example, [if a `PsiClass` doesn't have a `containingClass` it cannot be an anonymous class, therefore it should always have a `qualifiedName`](https://github.com/minecraft-dev/MinecraftDev/pull/123/files#diff-2f51dab0b9d851253f3671e2a778ecc0R29).